maths/Kynea_numbers.dart

```diff
@@ -0,0 +1,37 @@
+// CPP code to find nth Kynea number
+
+//#include <bits/stdc++.h>
+//using namespace std;
```

Comment:
That's unexpected!

Reply:
Sir, I am aware of C++ and learning Dart. I won't repeat this mistake in future.
Sorry Sir